xc7/utils/timing_utils.tcl

```diff
@@ -0,0 +1,109 @@
+proc write_timing_info {filename} {
```

Comment:
Maybe this is better in `xc7/utils/vivado/timing_utils.tcl`

Reply:
I think we already an excessively deep folder hierarchy, I see no reason to make it even deeper, especially considering it would have at most 4 items, and is joining 14 other items.